tests/test_core.py

```diff
@@ -871,9 +872,15 @@ class TestConfig(unittest.TestCase):
     # pylint: disable=invalid-name
     def setUp(self):
         """Set up things to be run when tests are started."""
-        self.config = ha.Config()
+        self.hass = get_test_home_assistant()
```

Comment:
Now all tests in this class are getting a HA instance created.

Reply:
OK, fixed now.